packages/flutter/lib/src/widgets/routes.dart

```diff
@@ -1378,6 +1378,21 @@ abstract class ModalRoute<T> extends TransitionRoute<T> with LocalHistoryRoute<T
   ///  * [ModalBarrier], the widget that implements this feature.
   String? get barrierLabel;
 
+  /// {@template flutter.widgets.ModalRoute.barrierOnTapHint}
+  /// The semantic hint text that informs users what will happen if they
+  /// tap on the widget.annouced in the format of 'Double tap to ...'.
+  ///
+  /// If the field is null, the default hint will be used, which results in
+  /// announcement of 'Double tap to activate'.
+  /// {@endtemplate}
+  ///
+  /// See also:
+  ///
+  ///  * [barrierDismissible], which controls the behavior of the barrier when
+  ///    tapped.
+  ///  * [ModalBarrier], the widget that implements this feature.
+  String? get barrierOnTapHint => null;
```

Comment:
Why is this defined in this class, it seems it can be in bottom sheet instead.

Reply:
Thanks! Moved to ModalBottomSheetRoute.